hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncConnectionImpl.java

```diff
@@ -79,7 +79,7 @@ public class AsyncConnectionImpl implements AsyncConnection {
 
   private static final Logger LOG = LoggerFactory.getLogger(AsyncConnectionImpl.class);
 
-  static final HashedWheelTimer RETRY_TIMER = new HashedWheelTimer(
+  public static final HashedWheelTimer RETRY_TIMER = new HashedWheelTimer(
```

Comment:
Better not make this public. ZKConnectionRegistry is in the same package, so we do not need to expose this as public?